src/services/api-key-management/api-key.service.ts

```diff
@@ -32,13 +34,14 @@ export class ApiKeyService {
     findOneById(id: number): Promise<ApiKey> {
```

Comment:
You could argue that "findOneById" should change name to "findOneByIdWithPermissions".